package-lock.json

```diff
@@ -4664,7 +4664,7 @@
 					}
 				},
 				"prettier": {
-					"version": "npm:prettier@2.2.1-beta-1",
+					"version": "npm:wp-prettier@2.2.1-beta-1",
```

Comment:
@fluiddot should this change be included? I have seen it occur before.

Reply:
I will fix the issue locally and update the PR.

Reply:
Nope, we shouldn't add it. I'd recommend you to run `nm ci` instead of `npm install` when installing dependencies to prevent this issue.